plugins/LadspaEffect/caps/interface.cc

```diff
@@ -58,18 +56,11 @@
 #define N 39 
 static DescriptorStub * descriptors [N];
 
-/*static inline void
-seed()
-{
-	static struct timeval tv;
-  gettimeofday (&tv, 0);
-
-	srand (tv.tv_sec ^ tv.tv_usec);
-}*/
-
 extern "C" {
 
-__attribute__ ((constructor)) 
+#ifdef __GNUC__
+	__attribute__((constructor))
+#endif
```

Comment:
I think it will make this function called twice when compiled with GCC.

Reply:
There was no guards before anyway. Even before adding the `#ifdef __GNUC__`, the function was being called twice. Should i add a guard here?

Reply:
`__attribute__((constructor))` is a GCC extension which causes this function to be called on module initialization. Since we have a separate `struct` for initialization, I think you should drop this completely.

Reply:
done